### ISSUE TITLE
KMP / JS performance issue when setting class property

### ISSUE THREAD

**Issue by nicolidin** [nicolidin/BrainstormAboutAnything#1]
Hi @avwie, I've seen your post on youtrack https://youtrack.jetbrains.com/issue/KT-52044/KJS:-Large-performance-penalty-when-using-Element.setAttribute-i  about some performance issue, and as I said in the comments I'm facing the same problem...
Have you fond some way to avoid it ?
I just can't understand there are such blocking bugs in KMP...
Maybe there is a beta (dev) version that allow to prevent such drastic performance issue..

**Comment by avwie** [comment]
Yes, when using element.asDynamic().setAttribute it is more stable

**Comment by nicolidin** [comment]
Oh yes I see, however I can't do such things because I'm facing the issue on generated javascript (transpiled from kotlin).
I think I'll have to wait, they fix it !

**Comment by nicolidin** [comment]
Yes I see, however I can't do that in commonKotlin, you can cause you are in kotlin-js.

**Comment by avwie** [comment]
I am not entirely sure this is the same issue. I am not sure if it is because of the extra method call. But in my case in the extra method call there is an instanceof call, which is expensive and unnecessary. I see in your flamegraph that the set method takes a long time. Do you have the dump of the performance test?
